.github/workflows/molecule.yaml

```diff
@@ -42,7 +42,6 @@ jobs:
         - edpm_sshd
         - edpm_ssh_known_hosts
         - edpm_timezone
-        - edpm_telemetry
```

Comment:
This is intentional?

Reply:
Found a comment on slack about this. It is intentional.

Reply:
Yes, this would exclude edpm_telemetry role from being testing in github workflows. There's an issue while doing this. While running Kepler container inside the ubi container it reported this error 
```
error removing memlock: failed to set memlock rlimit: operation not permitted
```
So I have moved testing this role into VM by making change in `zuul.d/jobs.yaml`